BAT/BCPriorModel.h

```diff
@@ -41,10 +41,6 @@ class BCPriorModel : public BCModel
      * @param call_likelihood Flag to control calling of Model's likelihood. */
     BCPriorModel(BCModel& model, bool call_likelihood = false);
 
-    /**
-     * destructor. */
-    virtual ~BCPriorModel() {};
```

Comment:
I guess here it's ok to leave it out and the implicitly generated dtor is virtual and empty.